examples/test-client/src/components/Navbar.tsx

```diff
@@ -20,7 +20,7 @@ export function Navbar({ connecting }: { connecting: boolean }): ReactNode {
   return (
     <div className="navbar bg-base-100 w-full">
       <div className="flex-none">
-        <a className="btn btn-ghost text-lg md:text-xl">Zupass API Client</a>
+        <a className="btn btn-ghost text-lg md:text-xl">Parcnet API Client</a>
```

Comment:
nit: i've only seen it written in all caps so far
```suggestion
        <a className="btn btn-ghost text-lg md:text-xl">PARCNET API Client</a>
```

Reply:
in any case, we should do a round of copy review before devcon for all the example zapps